changelogs/fragments/10299-github_app_access_token-lookup.yml

```
@@ -0,0 +1,4 @@
+bugfixes:
```

This is a feature, not a bugfix.
```suggestion
minor_changes:
```